### PR TITLE
PDI-1513: Add pingone_branding_theme_default (Resource) Export

### PR DESCRIPTION
**Pull request by erikostien-pingidentity**

### CODE REVIEW

File: internal/connector/pingone_platform/resources/pingone_branding_theme_default.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+			var themeName *string
+			var themeNameOk = false
+			if themeConfigurationOk {
+				themeName, themeNameOk = themeConfiguration.GetNameOk()
+			}
+
+			if themeConfigurationOk && themeNameOk {
+				importBlocks = append(importBlocks, connector.ImportBlock{
+					ResourceType: r.ResourceType(),
+					ResourceName: fmt.Sprintf("%s_active", *themeName),
```
Comment (henryrecker-pingidentity) [inline internal/connector/pingone_platform/resources/pingone_branding_theme_default.go:67]: Do we need the API calls for this resource?

Comment (erikostien-pingidentity) [reply internal/connector/pingone_platform/resources/pingone_branding_theme_default.go:67]: I'm not sure how else to determine if the theme is the "default". The import blocks of the themes in the previous resource do not retain this information.

Comment (erikostien-pingidentity) [reply internal/connector/pingone_platform/resources/pingone_branding_theme_default.go:67]: Unless we don't mind which name we attribute to the default resource in this case...

Comment (henryrecker-pingidentity) [reply internal/connector/pingone_platform/resources/pingone_branding_theme_default.go:67]: Yeah, I don't think it's super important to give it the specific name of the theme, personally.

Comment (erikostien-pingidentity) [reply internal/connector/pingone_platform/resources/pingone_branding_theme_default.go]: Sounds good. removed!

**Review by henryrecker-pingidentity — APPROVED** [review]

**Pull request merged**